android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/facts/FxSuggestFactsMiddleware.kt

```diff
@@ -58,7 +62,12 @@ class FxSuggestFactsMiddleware : Middleware<BrowserState, BrowserAction> {
                     FxSuggestSuggestionProvider.MetadataKeys.IMPRESSION_INFO,
                 ) as? FxSuggestInteractionInfo
                 impressionInfo?.let {
-                    emitSuggestionImpressedFact(it, positionInGroup, isClicked = isClicked)
```

Comment:
I fixed this up while I was here—it should've been `positionInAwesomeBar`, not `positionInGroup` 😬